src/renderer/helpers/api/local.js

```diff
@@ -437,3 +449,27 @@ export function mapLocalFormat(format) {
     url: format.url
   }
 }
+
+/**
+ * @param {import('youtubei.js/dist/src/parser/classes/comments/Comment').default} comment
+ */
+export function parseLocalComment(comment, hasOwnerReplied = false, replyToken = null) {
+  return {
+    dataType: 'local',
+    authorLink: comment.author.id,
+    author: comment.author.name,
+    authorThumb: comment.author.best_thumbnail.url,
+    isPinned: comment.is_pinned,
+    isOwner: comment.author_is_channel_owner,
+    isMember: comment.is_member,
+    text: Autolinker.link(parseLocalTextRuns(comment.content.runs, 14)),
```

Comment:
Are custom emojis still supported?

Reply:
See pinned comment for example: 
https://youtu.be/erVgWScY9Q4

Reply:
Yes (discussed in Matrix channel)